Sources/NIOSSL/TLSConfiguration.swift

```diff
@@ -200,7 +233,27 @@ public struct TLSConfiguration {
 
     /// The pre-TLS1.3 cipher suites supported by this handler. This uses the OpenSSL cipher string format.
     /// TLS 1.3 cipher suites cannot be configured.
-    public var cipherSuites: String
+    internal var cipherSuiteFormattedString: String = defaultCipherSuites
```

Comment:
We cannot hide `public var cipherSuites: String`: it's part of our stable API.

Reply:
Great point.  I believe this was caught in the automated check below as well.  I will address.

Reply:
This has been addressed in https://github.com/apple/swift-nio-ssl/pull/290/commits/a803d27261ad7f05a4377ece263eaff064107997